test/dds/communication/CMakeLists.txt

```diff
@@ -89,8 +90,6 @@ get_win32_path_dependencies(DDSSimpleCommunicationMonitor TEST_FRIENDLY_PATH)
 #  types test
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/simple_communication.py
     ${CMAKE_CURRENT_BINARY_DIR}/simple_communication.py COPYONLY)
-configure_file(${CMAKE_CURRENT_SOURCE_DIR}/example_type.xml
-    ${CMAKE_CURRENT_BINARY_DIR}/example_type.xml COPYONLY)
```

Comment:
In the new version of the test, the type is set in the constructor of each participant using the types generated with FastDDSGen, avoiding the use of Dynamic Types. I think we could do one of these options: 1. Keep the code as it is, 2. Replace it so that a Dynamic Type is generated from the XML, 3. Add the two options so that one of them can be chosen from an input argument.

I don't know if 3 is really interesting as the test is designed, but I could add a separate test to test the types generated in each of the ways.

Reply:
Lets keep it as it is right now!